examples/Introducing_CivisML_v2.ipynb

```diff
@@ -1059,15 +1072,16 @@
    "metadata": {},
    "source": [
     "## Hyperparameter optimization with Hyperband and Neural Networks\n",
-    "Multilayer Perceptrons (MLPs) are simple neural networks, which are now built in to CivisML. The MLP estimators in CivisML come from [muffnn](https://github.com/civisanalytics/muffnn), another open source package written and maintained by Civis Analytics. Let's fit one using hyperband.\n",
+    "Multilayer Perceptrons (MLPs) are simple neural networks, which are now built in to CivisML. The MLP estimators in CivisML come from [muffnn](https://github.com/civisanalytics/muffnn), another open source package written and maintained by Civis Analytics using [tensorflow](https://www.tensorflow.org/). Let's fit one using hyperband.\n",
     "\n",
-    "Hyperband is an alternative to grid search for hyperparameter optimization (learn about the details in [the original paper, Li et al. (2016)](https://arxiv.org/abs/1603.06560)). Grid search runs every parameter combination for the full time, while hyperband runs many combinations for a short time, then filters out the best, runs them for longer, filters again, and so on. This means that you can try more combinations in less time, so it's recommended to use it where possible. The hyperband estimator is open source and [available on GitHub](https://github.com/civisanalytics/civisml-extensions).\n",
+    "Tuning hyperparameters is a critical chore for getting an algorithm to perform at its best, but it can take a long time to run. Using CivisML 2.0, we can use hyperband as an alternative to conventional grid search for hyperparameter optimization-- it runs about twice as fast. While grid search runs every parameter combination for the full time, hyperband runs many combinations for a short time, then filters out the best, runs them for longer, filters again, and so on. This means that you can try more combinations in less time, so we recommend using it whenever possible. The hyperband estimator is open source and [available on GitHub](https://github.com/civisanalytics/civisml-extensions). You can learn about the details in [the original paper, Li et al. (2016)](https://arxiv.org/abs/1603.06560).\n",
```

Comment:
Only a comment -- Given unlimited computational resources, I think that grid search is actually faster. That's completely parallelizable, while hyperband has serial steps. I don't think that needs to change the text here, though.